createdisk.sh

```diff
@@ -11,19 +11,12 @@ OC=${OC:-oc}
 DEVELOPER_USER_PASS='developer:$2y$05$paX6Xc9AiLa6VT7qr2VvB.Qi.GJsaqS80TR3Kb78FEIlIL0YyBuyS'
```

Comment:
> Right now, if a user try to use master branch he is hoping get the
cluster deployed without any image override but this is not the case
right now since `git describe --exact-match --tags HEAD` provide latest
released tag and the image override happen.

This is only true if current git master HEAD is a tagged commit. 
```
[teuf@natto snc ((4.1.13))]$ LC_ALL=C git describe --exact-match --tags 
4.1.13
[teuf@natto snc ((4.1.13))]$ git checkout HEAD^

[teuf@natto snc ((83556dc...))]$ LC_ALL=C git describe --exact-match --tags 
fatal: no tag exactly matches '83556dccfa7dc208b3357c3e050dc559d7ef0657'
```
Since issues related to this have come up a few times recently, I'm not sure this git tag versioning is what we want, as apparently you want to be able to build both releases (which for now are tagged) and snapshots from the same snc commit. So maybe it's time to retire this git tag code and replace it with something else? Or maybe keep the tag versioning if this has benefits, and add a --snapshot argument (or env var)?